testkit-backend/src/main/java/neo4j/org/testkit/backend/channel/handler/TestkitRequestResponseMapperHandler.java

```diff
@@ -50,6 +46,16 @@ public void write(ChannelHandlerContext ctx, Object msg, ChannelPromise promise)
         ctx.writeAndFlush(responseStr, promise);
     }
 
+    @Override
+    public void exceptionCaught(ChannelHandlerContext ctx, Throwable cause) throws Exception {
```

Comment:
May I suggest moving this to `TestkitRequestProcessorHandler` please?

Reply:
Ah yes, that's much cleaner separation of concerns. I just didn't expect that a ChannelHandler further down the pipeline can handle errors that occurred earlier in the pipeline.